_includes/devdoc/guide_dash_features.md

```diff
@@ -110,6 +109,100 @@ had already been confirmed to a block depth of 5 in the blockchain.
 
 {% endautocrossref %}
 
+#### LLMQ InstantSend
+{% include helpers/subhead-links.md %}
+
+{% autocrossref %}
+
+The introduction of Long-Living Masternode Quorums in Dash Core 0.14 provides
+a foundation to further scale InstantSend. LLMQ-based InstantSend removes a
+number of previously required limitations and simplifies the process by decreasing
+the number of P2P messages clients must use.
+
+During the evaluation and transition from standard InstantSend to LLMQ-based
+InstantSend, Sporks 2 (`SPORK_2_INSTANTSEND_ENABLED`) and 20 (`SPORK_20_INSTANTSEND_LLMQ_BASED`)
+will both be used. Spork 2 enables or disables the entire InstantSend feature,
+while spork 20 determines which of the two InstantSend mechanisms is active when
+InstantSend is enabled.
+
+There are still some limitations on LLMQ-based InstantSend transactions:
+
+* Transaction inputs must either:
+  * Be in a block that has a ChainLock
```

Comment:
```suggestion
    * Be locked by InstantSend
    * Be in a block that has a ChainLock
```

Reply:
I see it's addressed below, but it makes sense to be in this list. Might want to remove or change line 144 and add this?